sympy/printing/tests/test_repr.py

```diff
@@ -101,12 +101,11 @@ def test_Rational():
 
 
 def test_Float():
-    sT(Float('1.23', prec=3), "Float('1.22998', prec=3)")
```

Comment:
>  no precision in lost between translations from decimal to binary

Right, so that confirms why the extra 3 digits were there before.  Thanks for this.

> Since now the translation from decimal to binary precision is taken care

But I'm not sure the conclusion follows (that the extra digits are now unnecessary).  although I've yet to find a counterexample.